res/css/views/auth/_AuthHeader.scss

```diff
@@ -20,6 +20,7 @@ limitations under the License.
     width: 206px;
     padding: 25px 25px;
     box-sizing: border-box;
+    background-color: white!important;
```

Comment:
Hi Samridhi,  please review how we have been implementing branding (e.g. https://dev.azure.com/ACTGovernment/Pegacorn/_workitems/edit/219811).  As we want to be able to push our fork back to element at some stage (to minimise merge issues when we pull from element master when required) we can't change the default behaviour of the code.  Instead either implement the change via:
1. The "Pegacorn communicate - Light theme custom colors" custom theme
2. Controlled by a config.json configuration property that defaults to the current value in the element code, if the propoerty is not in config.json
3. We have our own copy of the file in aether-host-files/common/element-web/matrix-react-sdk (this is the least favoured solution as it doesn't address the merge issues with element master code, but has been done for a quick tactical solution)

Reply:
![image](https://user-images.githubusercontent.com/12698292/116013333-0cbb8d80-a673-11eb-9f67-cb02c610dddc.png)
I just went through changes from Surendra on the changeset. I am able to see his changes on the same file.

Reply:
Yes but that file is in a different git repository, which is option 3 I described above.  If you implement your changes in that repo and revert the changes in this repo, that will be fine.

Reply:
Thanks, I have reverted this specific change as it was already a part of aether host files.